packages/common-types/src/baseTypes/aggregateTypes.ts

```diff
@@ -273,8 +273,8 @@ export interface ITKeyApi {
   outputShare(shareIndex: BNString, type?: string): Promise<unknown>;
   inputShareStore(shareStore: ShareStore): void;
   deleteShare(shareIndex: BNString): Promise<DeleteShareResult>;
-  encrypt(data: Buffer): Promise<EncryptedMessage>;
-  decrypt(encryptedMesage: EncryptedMessage): Promise<Buffer>;
+  encrypt(data: Buffer, keyType: KeyType): Promise<EncryptedMessage>;
+  decrypt(encryptedMesage: EncryptedMessage, keyType: KeyType): Promise<Buffer>;
```

Comment:
do need the keyType here, we use the tkey.keyType

Reply:
This is Tkey interfaces

Reply:
because the hashing of the key has been moved internally here.